samples/wrapper/strlcat.c

```diff
@@ -0,0 +1,57 @@
+/*	$OpenBSD: strlcat.c,v 1.19 2019/01/25 00:19:25 millert Exp $	*/
```

Comment:
If you use already existing functions' implementations, this file is not needed.